.github/workflows/kb_sdk-actions.yml

```diff
@@ -0,0 +1,136 @@
+name: Basic kb-sdk tests
```

Comment:
FWIW I usually try to do massive whitespace changes in their own PR. Makes the line count for a PR with actual content less scary.

Reply:
Yeah, I tried to put them all in the first commit to make it less frightening. Who are these monsters who use tabs and leave trailing spaces all over the place anyway?!